srcpkgs/kmplot/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'kmplot'
+pkgname=kmplot
+version=23.08.3
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="extra-cmake-modules gettext"
+makedepends="kdoctools-devel qt5-svg-devel kcompletion-devel kconfig-devel kguiaddons-devel kdbusaddons-devel kcrash-devel ki18n-devel kjobwidgets-devel kio-devel kparts-devel kiconthemes-devel kxmlgui-devel kservice-devel kwindowsystem-devel kitemviews-devel knewstuff-devel"
+depends="kparts hicolor-icon-theme"
+short_desc="Mathematical Function Plotter"
+maintainer="Luciogi <lucigithubcommit@skiff.com>"
+license="GPL-2.0-or-later, GFDL-1.2-or-later"
+homepage="https://apps.kde.org/kmplot"
+distfiles="${KDE_SITE}/release-service/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=455b9bbb195f89d1c0b978ca736e6862248e28812e1cccf4adcb312b512a5cea
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-host-tools qt5-qmake kcoreaddons pkgconf kconfig kdoctools"
```

Comment:
You can just add them unconditionally. They are needed all the time, they are just implicitly pulled in by make depends when not cross compiling.

Reply:
As I have slow internet, and HDD, it at bit faster to get dependencies and test locally

Reply:
Some non-representative benchmarks :)

The difference for native compiling on my system is quite small and bigger in userland than system.
If the difference were IO Heavy (like when Installing more when cross compiling), it would increase the time spent in kernel/system code)

Always in hostmakedepends
```
Benchmark 1: ./xbps-src -t extract kmplot
  Time (mean ± σ):     14.552 s ±  0.209 s    [User: 7.221 s, System: 5.669 s]
  Range (min … max):   14.229 s … 14.929 s    10 runs
```
Only in hostmakedepends
```
Benchmark 1: ./xbps-src -t extract kmplot
  Time (mean ± σ):     14.627 s ±  0.578 s    [User: 7.165 s, System: 5.730 s]
  Range (min … max):   14.298 s … 16.229 s    10 runs
```
Cross compiling
```
Benchmark 1: ./xbps-src extract kmplot -j32 -t -a aarch64-musl
  Time (mean ± σ):     16.848 s ±  0.758 s    [User: 6.684 s, System: 6.974 s]
  Range (min … max):   16.045 s … 18.587 s    10 runs
```

Reply:
ok I am removing if statement 😀